packages/Webkul/Shop/src/Resources/views/layouts/header/index.blade.php

```diff
@@ -77,7 +77,7 @@ class="search-field"
                             <i class="icon compare-icon"></i>
                             <span class="name">
                                 {{ __('shop::app.customer.compare.text') }}
-                                <span class="count">(<span id="compare-items-count"></span>)<span class="count">
```

Comment:
!important 
Opening tag is duplicated into closing tag.
I've tested the outcome of the fixed version, seems ok to me.